vmServer/send/post_request.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python
+"""Program to send a post request with
+  the proto file and accept and
+  save the response in response.pb
+
+  It takes the input_request.pb in the .\\proto\\ directory
+  sends it to the VM server, and accepts the response as
+  response.pb and saves it in .\\proto\\ directory
+"""
+import os
+import requests
+
+
+URL = "http://127.0.0.1:5000/load"
+ROOT = ".\\proto\\"
```

Comment:
Is there a reason for the convention to use fully capitalized variable names here? It's better to be consistent throughout.

Reply:
also a couple of spacing issues in the file

Reply:
Space before and after operator has been added

Reply:
As we have discussed, it is a constant, we decided to leave it in capital letters